steembase/memo.py

```diff
@@ -144,7 +144,7 @@ def decode_memo(priv, message):
     message = aes.decrypt(unhexlify(bytes(message, 'ascii')))
     try:
         return _unpad(message.decode('utf8'), 16)
-    except:
+    except:  # noqa FIXME(sneak)
```

Comment:
oy

Reply:
it was there when i got here. just making it pass linting so that nobody can *add* shit like this.

Reply:
oh, i know, you are the bearer of the news, not its author